.github/workflows/update-license-file.yml

```diff
@@ -0,0 +1,35 @@
+name: Update LICENSE file
```

Comment:
Please update the commit message because I think that you made a small mistake. Something like:

Add workflow to automatically update LICENSE file with recipe licenses